src/wat-writer.cc

```diff
@@ -122,14 +122,14 @@ class WatWriter {
   Result result_ = Result::Ok;
   int indent_ = 0;
   NextChar next_char_ = NextChar::None;
-  int depth_ = 0;
+  Index depth_ = 0;
```

Comment:
Depth is an index into the label index space, see: http://webassembly.github.io/spec/syntax/modules.html#indices